actionview/lib/action_view/helpers/number_helper.rb

```diff
@@ -29,7 +29,7 @@ def initialize(number)
       # Additionally, supports a +:raise+ option that will cause
       # InvalidNumberError to be raised if +number+ is not a valid number:
       #
-      #   number_to_phone("12x34")              # => "12x34"
+      #   number_to_phone("1234567890")         # => "123-456-7890"
```

Comment:
These examples show that passing a invalid number just returns the input by default.
So I think the examples are correct.

We can add a valid example after the opening line.
```markdown
# Delegates to ActiveSupport::NumberHelper#number_to_phone.
#
#   number_to_phone("1234567890") # => "123-456-7890"
#
```